internal/server/count/count.go

```diff
@@ -132,3 +128,69 @@ func Count(
 	}
 	return result, nil
 }
+
+// Count checks if entities have data for stat vars and stat var groups.
+//
+// Returns a two level map from sv/svg dcid to entity dcid to the number of
+// sv with data. For a given sv/svg, if an entity has no data, it will
+// not show up in the second level map.
+func Count(
+	ctx context.Context,
+	st *store.Store,
+	cachedata *cache.Cache,
+	svOrSvgs []string,
+	entities []string,
+) (map[string]map[string]int32, error) {
+	result, err := countInternal(ctx, st, cachedata, svOrSvgs, entities)
+	if err != nil {
+		return nil, err
+	}
+	// Check for count for computed observations.
+	// Use counts of formula variables as a heuristic.
+	for _, dcid := range svOrSvgs {
+		formulas, ok := cachedata.SVFormula()[dcid]
+		if !ok {
+			continue
+		}
+		missingEntities := []string{}
+		for _, entity := range entities {
+			if _, ok := result[dcid][entity]; !ok {
+				missingEntities = append(missingEntities, entity)
+			}
+		}
```

Comment:
If the svOrSvg is a computed one, no entities would have the count data from countInternal? Should the following count happens when len(missingEntities) == len(entities)?

Reply:
Spoke offline - updated to check for all formulas in one call